src/environment.ts

```diff
@@ -41,6 +41,9 @@ export enum EnvVar {
   /** Whether the CodeQL Action has already warned the user about low disk space. */
   HAS_WARNED_ABOUT_DISK_SPACE = "CODEQL_ACTION_HAS_WARNED_ABOUT_DISK_SPACE",
 
+  /** Whether the init action has been run. */
+  INIT_ACTION_HAS_RUN = "CODEQL_INIT_ACTION_HAS_RUN",
```

Comment:
Minor: Doesn't read as well but it's nice to have everything the Action defines prefixed by `CODEQL_ACTION_`

```suggestion
  INIT_ACTION_HAS_RUN = "CODEQL_ACTION_INIT_HAS_RUN",
```